### PR TITLE
Replace Guava primitive compare() with JDK7+ compare()

### PR DESCRIPTION
**Pull request by msgilligan**
In Java 7, Integer.compare() and Long.compare() were added. Use those
methods rather than the Guava equivalents which are “deprecated” for
JDK7+.

### CODE REVIEW

File: core/src/main/java/org/bitcoinj/core/Sha256Hash.java [hunk 1]
```diff
@@ -18,7 +18,7 @@
 package org.bitcoinj.core;
 
 import com.google.common.io.ByteStreams;
-import com.google.common.primitives.*;
```
Comment (schildbach) [inline core/src/main/java/org/bitcoinj/core/Sha256Hash.java]: This is an unrelated change.

Comment (msgilligan) [reply core/src/main/java/org/bitcoinj/core/Sha256Hash.java]: It's somewhat related, since I'm trying to identify places in the code that are using Guava functions that are now provided by the JDK. When I looked at this file, I saw it was using `Ints` (only) but not for the methods I was replacing. Do you want me to remove it and send it separately?

**Comment by schildbach** [comment]
I merged it without the change in Sha256Hash. I wouldn't worry about the import unless you want to change the usage of that import too.

At one point we could "organize imports" over the whole project, but I never bothered enough.
